### DISCUSSION TITLE
GitHub issues

### DISCUSSION THREAD

**Discussion by Bghosh-2024** [community/community#132634] [New to GitHub]
When I checked out to main and added the following command, it shows an error message:

git checkout main
git pull upstream main

 

Does this mean I have a problem with me repo? Please let me know.

Note: I can create the branches and the PR for respective tickets.

![setupstream_issue_GitHub](https://github.com/user-attachments/assets/b9b95802-8576-41e1-a612-10f0b73511e9)

Here are the details:

My fork: https://github.com/Bghosh-2024/Bghosh-2024
git clone https://github.com/Bghosh-2024/xandr-docs-pr.git

**Comment by Kolophonium0** [comment]
Did you set upstream? Try: `git remote -v` If there's no upstream in the output try: `git remote add upstream ...`

**Reply by Kolophonium0** [reply]
This may help you: https://www.atlassian.com/git/tutorials/git-forks-and-upstreams

**Reply by Bghosh-2024** [reply]
Thank you.

**Reply by ghostinhershell** [reply]
Hi @Bghosh-2024 👋🏾, 

It looks like @Kolophonium0's response was the answer to your question here. Can you please mark their response as the answer here? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![Screenshot of a comment marked as the answer to a discussion.](https://github.com/user-attachments/assets/00044ad0-409b-48ca-b597-5a7014774099)

**Comment by Bghosh-2024** [comment]
This link was really useful: 
https://www.atlassian.com/git/tutorials/git-forks-and-upstreams

I found out that I have two repositories:

xandr-docs-pr (Private)
Bghosh-2024 (Public)
My working repository is: "MicrosoftDocs/xandr-docs-pr".

So, I navigated to my cloned repository and followed the steps below:

> cd c:/GitHub/xandr-docs-pr
> git remote remove upstream (removed the incorrect upstream remote)
> git remote add upstream https://github.com/MicrosoftDocs/xandr-docs-pr.git 

(added the correct upstream remote)

> git remote -v (verified the remotes)

The following messages appeared:

- origin https://github.com/Bghosh-2024/xandr-docs-pr.git (fetch)
- origin https://github.com/Bghosh-2024/xandr-docs-pr.git (push)
- upstream https://github.com/MicrosoftDocs/xandr-docs-pr.git (fetch)
- upstream https://github.com/MicrosoftDocs/xandr-docs-pr.git (push)

Then, I executed the following commands:

> git fetch upstream
> git checkout main
> git pull upstream main

This time it worked!

**Comment by Tusho7** [comment]
You have to try `git remote -v` .

I think it will help you.
